networking/dns/dns-operator.adoc

```diff
@@ -0,0 +1,19 @@
+[id="dns-operator"]
+= DNS Operator in {product-title}
+include::modules/common-attributes.adoc[]
+:context: dns-operator
+
+toc::[]
+
+The DNS Operator deploys and manages CoreDNS to provide a name resolution
+service to pods, enabling DNS-based Kubernetes Service discovery in
+OpenShift. A Kubernetes `DaemonSet` is used by DNS Operator to manage
+the name resolution service.
```

Comment:
The last sentence seems a little indirect and redundant.  Simplified: "The operator deploys CoreDNS using a `DaemonSet`."  Alternatively, this paragraph could stick to the abstract, high-level description, omitting this sentence and the earlier mention of CoreDNS, and leave the details for `modules/nw-dns-operator.adoc`. 
 What do you think?

Reply:
From most abstract to least:

Cluster DNS enables DNS-based service discovery for pods
DNS Operator provides managed cluster DNS
CoreDNS is the underlying managed cluster DNS implementation

Thoughts?

Reply:
I simplified the intro with the following:

> The DNS Operator deploys and manages CoreDNS to provide a name resolution
service to pods, enabling DNS-based Kubernetes Service discovery in
OpenShift.

I figure keep it simple, to the point and let the rest of the docs expand upon the intro.